tests/system/test_system.py

```diff
@@ -93,7 +93,11 @@ def _list_instances():
 
 
 def setUpModule():
-    Config.CLIENT = Client()
+    if USE_EMULATOR:
+        from google.auth.credentials import AnonymousCredentials
+        Config.CLIENT = Client(project='emulator-test-project', credentials=AnonymousCredentials())
```

Comment:
I wonder can we read the project from an env var instead of using a hard-coded one?